### ISSUE TITLE
Does not compile with stackage lts-7 & ghc-8

### ISSUE THREAD

**Issue by alexanderkjeldaas** [RaphaelJ/friday#26]
```
cat /root/ocr/learn/learn/.stack-work/logs/friday-0.2.2.0.log

Configuring friday-0.2.2.0...
setup-Simple-Cabal-1.24.0.0-ghc-8.0.1: Encountered missing dependencies:
transformers >=0.3 && <0.5 && ==0.5.2.0
```

**Comment by alexanderkjeldaas** [comment]
Now it doesn't compile with lts-7.

I'm changing the title of this ticket.

**Comment by alexanderkjeldaas** [comment]
```In the dependencies for friday-0.2.2.0:
    ratio-int must match >=0.1.2 && <0.2, but the stack configuration has no specified version
              (latest applicable is 0.1.2)
    transformers-0.5.2.0 must match >=0.3 && <0.5 (latest applicable is 0.4.3.0)```

**Comment by alexanderkjeldaas** [comment]
#27 solves this it seems.

**Comment by TomMD** [comment]
I merged #27 so will close this ticket.  Thanks for the report and persistence.

**Comment by alexanderkjeldaas** [comment]
@TomMD thank you!
